packages/docusaurus/lib/webpack/loaders/markdown/index.js

```diff
@@ -6,12 +6,16 @@
  */
 
 const fm = require('front-matter');
+const mdx = require('@mdx-js/mdx');
 const {getOptions} = require('loader-utils');
 const path = require('path');
 const {resolve} = require('url');
 
-module.exports = function(fileString) {
-  const options = getOptions(this);
+module.exports = async function(fileString) {
```

Comment:
Make this asynchronous loader

Reply:
Interesting, I didn't know this was possible.